lib/timers.js

```diff
@@ -94,8 +94,8 @@ const TIMEOUT_MAX = 2147483647; // 2^31-1
 //
 // - key = time in milliseconds
 // - value = linked list
-const refedLists = {};
-const unrefedLists = {};
+const refedLists = Object.create(null);
```

Comment:
I understand this is not about performance, but just out of curiosity, I measured the performance difference.
 
`console.time("{}");
for (var i = 100000; i--;) {
  var a = {};
}
console.timeEnd("{}");
var create = Object.create;
console.time("create");
for (var i = 100000; i--;) {
  var b = create(null);
}
console.timeEnd("create");
`

`{}: 5.792ms`
`create: 15.221ms`

Probably, calling a function is heavier than {}